src/olmo_core/launch/beaker.py

```diff
@@ -313,7 +322,10 @@ def beaker(self) -> Beaker:
         The Beaker client.
         """
         if self._beaker is None:
-            self._beaker = Beaker.from_env(default_workspace=self.workspace)
+            kwargs = {}
+            if self.workspace:
```

Comment:
Does no workspace mean use the user's default ws?

Reply:
Right